src/itree.cpp

```diff
@@ -31,13 +38,17 @@ auto print_dir(const fs::path&,
 auto correct_path(const po::variables_map&, rodsEnv&) -> fs::path;
 auto print_usage() -> void;
 auto get_json(const fs::path&, const po::variables_map&, ix::client_connection&, unsigned int depth) -> json::json;
+auto is_defaulted(const pattern_matcher& pm) -> bool {
```

Comment:
Consider renaming this function or using more words.

I found myself constantly revisiting the definition to figure out what `is_defaulted` meant.

Reply:
---